pkg/controllers/openshift-controller-manager.go

```diff
@@ -129,8 +124,21 @@ func (s *OCPControllerManager) Run(ctx context.Context, ready chan<- struct{}, s
 		klog.Fatalf("failed to apply openshift namespaces %v", err)
 	}
 
-	options := openshift_controller_manager.OpenShiftControllerManager{Output: os.Stdout}
-	options.ConfigFilePath = s.ConfigFilePath
+	clientConfig, err := helpers.GetKubeClientConfig(configv1.KubeClientConfig{KubeConfig: s.kubeconfig})
+	if err != nil {
+		return err
+	}
+
+	go func(ctx context.Context) error {
+
+		errorChannel <- openshift_controller_manager.RunOpenShiftControllerManager(&s.config, clientConfig)
+
+		select {
+		case <-ctx.Done():
+			klog.Infof("OpenShift Controller Manager received stop signal. exiting.")
+			return nil
+		}
+	}(ctx)
```

Comment:
Since RunOpenShiftControllerManager doesn't block, I don't think that calling it from this goroutine is providing any benefit over something like:

```suggestion
	if err := openshift_controller_manager.RunOpenShiftControllerManager(&s.config, clientConfig); err != nil {
		return err
	}
	<-ctx.Done()
	return nil
```

Reply:
Suggestion accepted